root/static/scripts/relationship-editor/common/entity.js

```diff
@@ -132,24 +135,19 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
                     }
 
                     if (ko.unwrap(group.canBeOrdered)) {
-                        var hasOrdering = group.values.any(function (r) {
-                            return r.linkOrder() > 0;
-                        });
+                        var hasOrdering = group.values.any(function (r) { return r.linkOrder() > 0 });
```

Comment:
This change seems incorrect, and so do the others on this file. Probably introduced when resolving conflicts.